workerapi/api.go

```diff
@@ -78,6 +96,8 @@ func DomainRunCommandToThrift(domain *runner.Command) *worker.RunCommand {
 	thrift.JobId = &jobID
 	taskID := domain.TaskID
 	thrift.TaskId = &taskID
+	tag := domain.Tag
+	thrift.Tag = &tag
```

Comment:
You can use copyString(domain.Tag) here right?

Reply:
nope - that fails workerapi/api_test.go

we expect the pointer value even for empty strings